libdnf5/module/module_sack.cpp

```diff
@@ -544,14 +547,14 @@ std::pair<std::vector<std::vector<std::string>>, ModuleSack::ModuleErrorType> Mo
 
     if (ret == libdnf5::GoalProblem::NO_PROBLEM) {
         set_active_modules(goal_weak);
-        return make_pair(problems, ModuleSack::ModuleErrorType::ERROR);
+        return make_pair(problems, GoalProblem::MODULE_SOLVER_ERROR);
     }
 
     auto logger = base->get_logger();
     logger->critical("Modularity filtering totally broken\n");
 
     active_modules.clear();
-    return make_pair(problems, ModuleSack::ModuleErrorType::CANNOT_RESOLVE_MODULES);
+    return make_pair(problems, GoalProblem::MODULE_SOLVER_ERROR);
```

Comment:
Note: I removed the distinction between the `ERROR` and the `CANNOT_RESOLVE_MODULES`, because I don't think it's used anywhere.